### ISSUE TITLE
BuildingsPy (or Travis) produces inconsistent results

### ISSUE THREAD

**Issue by Mathadon** [lbl-srg/BuildingsPy#208]
Sometimes not all unit tests are executed. E.g. in https://travis-ci.org/open-ideas/IDEAS/jobs/383240293 not all tests are executed, while they are executed correctly in https://travis-ci.org/open-ideas/IDEAS/jobs/382014283.

In the former case the travis log contains:

```
export TEST_ARG="make test-dymola    PACKAGE=IDEAS.Fluid.{HeatExchangers,HeatPumps,Interfaces,MassExchangers,MixingVolumes,Movers,Sensors,Sources,Storage}"
...
(cd ../../../../IDEAS && python ../bin/runUnitTests.py --batch --single-package IDEAS.Fluid.Storage --tool dymola)
```

instead of 

```
export TEST_ARG="make test-dymola    PACKAGE=IDEAS.Fluid.{HeatExchangers,HeatPumps,Interfaces,MassExchangers,MixingVolumes,Movers,Sensors,Sources,Storage}"
...
(cd ../../../../IDEAS && python ../bin/runUnitTests.py --batch --single-package IDEAS.Fluid.{HeatExchangers,HeatPumps,Interfaces,MassExchangers,MixingVolumes,Movers,Sensors,Sources,Storage} --tool dymola)
```

I.e. in the former case only the tests for `IDEAS.Fluid.Storage` are run.

This is using the master branch of BuildingsPy. Have you had similar issues before?

Thanks!

**Comment by mwetter** [comment]
@Mathadon : I have not seen this behavior. Can you reproduce it?
Note also the telephone icon on line 384 on https://travis-ci.org/open-ideas/IDEAS/jobs/383240293 Does this mean call IDEAS support?

**Comment by Mathadon** [comment]
@mwetter yes I can, see IBPSA master build: https://travis-ci.org/ibpsa/modelica-ibpsa/jobs/382206551 . Same problem. This makes me think that something is going on at travis..?

Sure not, there exists no such thing!

**Comment by mwetter** [comment]
Indeed. I verified the Makefile and cannot explain this behavior. Do you want to send an email to support@travis-ci.com?

**Comment by Mathadon** [comment]
@mwetter I think I found it. I'm not sure whether you know about brace expansion in shell scripts? See https://stackoverflow.com/questions/19291746/escape-curly-braces-in-unix-shell-script .

When I go to `IBPSA/Resources/Scripts/travis` and execute `make test-dymola    PACKAGE=IDEAS.{Media,Utilities}` then this also results 'incorrectly' in:

```
(cd ../../../../IDEAS && python ../bin/runUnitTests.py --batch --single-package IDEAS.Utilities --tool dymola)
Regression tests are only run for the following package:
  IDEAS.Utilities
``` 

you see what happens if you do 

```
echo make test-dymola    PACKAGE=IDEAS.{Media,Utilities}
>>> make test-dymola PACKAGE=IDEAS.Media PACKAGE=IDEAS.Utilities
```

what does work is:

```
make test-dymola    PACKAGE="IDEAS.{Media,Utilities}"
>>> (cd ../../../../IDEAS && python ../bin/runUnitTests.py --batch --single-package IDEAS.{Media,Utilities} --tool dymola)
```

My best guess is that travis updated some software that now interprets our code differently.. But I think that the 'bug' is on our side of the code. I'll add the quotes to verify.

**Comment by mwetter** [comment]
@Mathadon I think this is what is happening. Please make a pull request if quotes fix the problem
